draft-oid4vc-haip-sd-jwt-vc.md

```diff
@@ -356,17 +359,29 @@ The following is a non-normative example of a presentation definition for a SD-J
 
 # Crypto Suites
 
-Issuers, holders and verifiers MUST support P-256 (secp256r1) as a key type with ES256 JWT algorithm for signing and signature validation whenever this profiles requires to do so:
+Cryptography is required by the following operations:
+
+- Attestation of trust with the entities of the ecosystem.
+- Issuance and presentation using the cryptografic proof of possession of the Wallet Instance Attestations.
+- Issuance of the Verifiable Credentials.
+- Presentation Authorization requests: the process where a Verifier requests to the Holder the presentation of the Verifiable Credentials.
```

Comment:
I think we need to align in one term. The VP spec currently either uses Authorization Request or Presentation Request but nowhere Presentation Authorization Request. I think Presentation Request makes the most sense.

Reply:
aligned at time of your revision @tlodderstedt

Reply:
what do you mean? OID4VP still uses Presentation Request or Authorization Request